moviepy/utils.py

```diff
@@ -11,6 +11,25 @@
 
 
 def close_all_clips(objects="globals", types=("audio", "video", "image")):
+    """Closes all clips in a context.
+
+    Follows different strategies retrieving the namespace from which the clips
+    to close will be retrieved depending on the ``objects`` argument, and filtering
+    by type of clips depending on the ``types`` argument.
+
+    Parameters
+    ----------
+
+    objects : str/dict, optional
```

Comment:
```suggestion
    objects : str or dict, optional
```
According to https://numpydoc.readthedocs.io/en/latest/format.html#sections :)